chirpstack-concentratord-sx1302/src/config/mod.rs

```diff
@@ -110,6 +111,8 @@ pub struct Gateway {
     pub com_dev_path: Option<String>,
     pub i2c_dev_path: Option<String>,
 
+    #[serde(skip)]
+    pub gateway_id_bytes: Vec<u8>,
```

Comment:
It is better to make this a `Some([u8; 8]);`

Reply:
`Some([u8; 8])` is not a type, used `Option<[u8; 8]>` instead.